src/main/groovy/com/palantir/gradle/docker/DockerExtension.groovy

```diff
@@ -85,12 +87,25 @@ class DockerExtension {
         copySpec.from(files)
     }
 
-    public Set<String> getTags() {
-        return tags
+    public Set<String> getUnresolvedTags() {
```

Comment:
Renaming this method is an API break. Maybe rename the new method to something like `getNamedTags`?

Reply:
Ok, but I think we will delete the tags dsl finally.